server/lib/config/pbench-server-default.cfg

```diff
@@ -128,17 +128,37 @@ lowerbound = 820
 # uri = driver://user:password@hostname/dbname
 wait_timeout = 120
 
+[authentication]
+# Secret key for use by the Pbench Server API sub-system for security related
+# tasks.
+secret-key = my_precious
+
+# User authentication section to use when authorizing the user with an OIDC
+# identity provider.
+
+# URL of the OIDC auth server
+#server_url = http://keycloak-broker.example.com:8080
+
+# Realm name that is used for the authentication with OIDC
+realm = pbench
+
+# Client entity name requesting OIDC to authenticate a user
+client = pbench-client
+
+# Client secret if the above client is not public
+keycloak-secret = <keycloak_secret>
```

Comment:
This should be commented, since it's not an actual default.

Reply:
Addressed in PR #3081.

Reply:
_[Unresolving...]_

How was this (and the [previous](https://github.com/distributed-system-analysis/pbench/pull/3114#discussion_r1045779728)) resolved by #3081?

I believe that that PR made this section optional, but this PR is still providing it...and, therefore, these changes (assuming we want them...) should include proper comments.

Reply:
Fixed.